### PR TITLE
feat: Products page

### PR DESCRIPTION
**Pull request by Tulun**
**Motivation**
New products pages for marketing site. Updated navbar.

**Details**
Products page have been built on the old static site.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [x]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

File: src/assets/scss/components/_nav.scss [hunk 1]
```diff
@@ -3,46 +3,180 @@
...
+    width: 100%;
+    text-align: left;
+  }
+
+  .dropdown-item-mobile {
+    display: none;
+    @include breakpoint-down(tablet) {
+      display: block;
+      width: 100%;
+      // margin-right: 12px;
```
Comment (daywiss) [inline src/assets/scss/components/_nav.scss:161]: stray comment

File: src/kpi-options.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        </p>
+        <p class="products-body-info">
+          Treasurers use them to diversify risk by selling them, and only paying
+          out the tokens if success criteria are met.
+        </p>
+
+        <h2 class="products-body-title">How is it better?</h2>
+        <p class="products-body-info">
+          Centralized performance payouts are trusted and are not tradable
+          Unrestricted airdrops are often dumped and dilute token supply,
```
Comment (daywiss) [inline src/kpi-options.html:51]: there might be  a typo or missing period between these lines

File: src/kpi-options.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+          The psychological impact of a shared upside invigorates communities.
+        </p>
+        <p class="products-body-info">
+          They are secured by UMA’s optimistic oracle. Recipients know they will
+          pay out as expected.
+        </p>
+
+        <h2 class="products-body-title">Examples</h2>
+        <p class="products-body-info">
+          A new project uses KPI Options to bootstrap their new community. They
```
Comment (daywiss) [inline src/kpi-options.html:64]: new here is used twice, feels redundant

File: src/lsp.html [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+        <h2 class="products-body-title">Examples</h2>
+        <p class="products-body-info">
+          A development team wants to express a crypto dominance pair, and
+          creates a LSP that expresses the relative value of an asset compared
+          to the entire crypto marketcap.
+        </p>
+        <p class="products-body-info">
+          A booky launches a sports prediction market using the LSP contracts.
+          The long/short tokens each represent the liklihood of the outcome of a
```
Comment (daywiss) [inline src/lsp.html:63]: spelling of likelihood

File: src/optimistic-oracle.html [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+            src="assets/images/powered_by_uma.png"
+            alt="UMA_logo"
+        /></span>
+        <div class="shell">
+          <div class="hero__inner">
+            <h1>Optimistic Oracle</h1>
+
+            <h4>
+              The Optimistic Oracle is used to retrieve prices or other data and
+              put them on-chain. It has an inbuilt resolution system in case of
```
Comment (daywiss) [inline src/optimistic-oracle.html:21]: i think there should be a mention of how this contrasts with dvm, ie the OO is faster and cheaper, something to that effect, @Tommy1231232

File: src/range-tokens.html [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+        </p>
+        <p class="products-body-info">
+          DAOs that need to prepare their treasury by diversifying. This ensures
+          they can fund their operations and achieve their protocol objectives.
+        </p>
+
+        <h2 class="products-body-title">How is it better?</h2>
+        <p class="products-body-info">
+          DAOs can add stablecoins to their balance sheet without having to
+          conduct a token sale, which can cause immeidate sell pressure and
```
Comment (daywiss) [inline src/range-tokens.html:54]: sp: immeidate

**Review by daywiss — COMMENTED** [review]
looks good, couple typos and wording things

File: src/kpi-options.html (hunk 3 above)
Comment (Tulun) [reply src/kpi-options.html:64]: I didn't write the copy ;)

**Review by daywiss — APPROVED** [review]
looks good

**Review by Gamaranto — APPROVED** [review]
Code looks good. I haven't checked the copy, but maybe @Tommy1231232 can give a final look on that. 

I only have some minor comments on how the page looks. I haven't looked at the design, so feel free to ignore if you think they're not relevant.

- The text on the product pages is not vertically aligned:
  
CleanShot 2021-06-17 at 20 06 39@2x

- The hero of each product page has text left aligned, on mobile, but everything else on the page is center aligned: 
CleanShot 2021-06-17 at 20 09 57@2x

- The hover effect for the product tab is slightly different, from the other ones

**Pull request merged**